data/json/items/obsolete.json

```diff
@@ -13,6 +13,53 @@
     "weight": "8440 g",
     "volume": "5 L"
   },
+  {
+    "result": "makeshift_grenade",
+    "type": "recipe",
+    "obsolete": true
+  },
+  {
+    "id": "makeshift_grenade",
+    "copy-from": "grenade",
+    "type": "TOOL",
+    "name": { "str": "makeshift grenade" },
+    "description": "An improvised explosive device cobbled together from parts.  Use this item to pull the pin and light the fuse.  You will then have some amount of time before it explodes; throwing it would be a good idea.",
```

Comment:
```suggestion
    "description": "An improvised explosive device cobbled together from parts.  Use this item to pull the pin and light the fuse.  You will then have about 5 seconds before it explodes; throwing it would be a good idea.",
```